### ISSUE TITLE
r: Expects /usr/local/Library/ENV to still exit

### ISSUE THREAD

**Issue by DomT4** [Homebrew/homebrew-science#3839]
_From @K20shores on July 18, 2016 19:59_

### Bug reports:

After updating to the newest version of Homebrew, some commands that deal with a brewed version of R no longer work. Two versions of this problem can be found below:

1.) http://stackoverflow.com/questions/38441722/error-using-rscript
2.) http://stackoverflow.com/questions/38440350/sed-directory-not-found-when-running-r-with-e-flag

It seems that /usr/local/Library/ENV was moved in the most recent update, but scripts that expect this directory to exist were not updated. 1. is my post. The version of Homebrew that produced this at the time was Homebrew 0.9.9 (git revision 45bf; last commit 2016-07-18).

_Copied from original issue: Homebrew/brew#553_

**Comment by DomT4** [comment]
_From @scpeters on July 18, 2016 20:4_

See also #544

**Comment by ck37** [comment]
I am also running into this issue. I cannot build documentation for my R package due to not finding sed :/

```r
==> devtools::document(roclets=c('rd', 'collate', 'namespace'))

/usr/local/Cellar/r/3.3.1/R.framework/Resources/bin/R: line 193: /usr/local/Library/ENV/4.3/sed: No such file or directory
ERROR: option '-e' requires a non-empty argument

Exited with status 1.
```

**Comment by blindjesse** [comment]
Installing from source (uninstalling then reinstalling with -s option) seemed to work for me.

**Comment by ck37** [comment]
Thanks, yep that worked for me too

**Comment by DomT4** [comment]
Should be fixed after a `brew update && brew upgrade r`. Not sure where the Science team is, but didn't want to leave it broken for any longer so bumped it myself.

**Comment by Apeder** [comment]
This was frustrating. After following the instructions above, I also had to reinstall the R kernel following the instructions here: https://github.com/IRkernel/IRkernel

**Comment by ilovezfs** [comment]
@Apeder is that a component brew itself handles or something you install after brew r has been installed?

**Comment by Apeder** [comment]
AFAIK, it's something to install after R, not managed by brew.

**Comment by dpo** [comment]
@Apeder the brew formula for R doesn't cover IRkernel, which isn't even registered on CRAN. Are you saying that in your case, brew R stll wants /usr/local/Library/ENV to exist? What were the specific error messages?

**Comment by Apeder** [comment]
The error message was the same as reported here: http://stackoverflow.com/questions/38440350/sed-directory-not-found-when-running-r-with-e-flag

`/usr/local/bin/R: line 193: /usr/local/Library/ENV/4.3/sed: No such file or directory
ERROR: option '-e' requires a non-empty argument`

This error persisted after I had upgraded to Sierra and tried `brew reinstall -s r` several times. After I reinstalled the IRkernel, things started working again. To be clear, R reinstalled successfully, though Jupyter would still throw the error above when it tried to initialize.
